spec/google/gax/operation_spec.rb

```diff
@@ -33,7 +33,6 @@
 require 'google/gax/settings'
 require 'google/gax/constants'
 require 'google/protobuf/any_pb'
```

Comment:
That should be fine. But let us hold off this PR until we know the release date of grpc 1.1.